diff -r e3ae850caa1e Setup.in
--- a/Setup.in	Mon Nov 03 00:54:48 2014 +0000
+++ b/Setup.in	Sun Dec 07 11:11:30 2014 -0600
@@ -16,7 +16,7 @@
 JPEG = -ljpeg
 SCRAP = -lX11
 PORTMIDI = -lportmidi
-PORTTIME = -lporttime
+PORTTIME =
 FREETYPE = -lfreetype
 AVFORMAT = -lavformat 
 SWSCALE = -lswscale 
diff -r e3ae850caa1e config_unix.py
--- a/config_unix.py	Mon Nov 03 00:54:48 2014 +0000
+++ b/config_unix.py	Sun Dec 07 11:11:30 2014 -0600
@@ -188,7 +188,7 @@
         Dependency('JPEG', 'jpeglib.h', 'libjpeg', ['jpeg']),
         Dependency('SCRAP', '', 'libX11', ['X11']),
         Dependency('PORTMIDI', 'portmidi.h', 'libportmidi.so', ['portmidi']),
-        Dependency('PORTTIME', 'porttime.h', 'libporttime.so', ['porttime']),
+        Dependency('PORTTIME', 'porttime.h', 'libportmidi.so', ['portmidi']),
         FFMPEGDependency('AVFORMAT', 'libavformat/avformat.h', 'libavformat.a', ['avformat'], ['/include', '/include/ffmpeg']),
         FFMPEGDependency('SWSCALE', 'libswscale/swscale.h', 'libswscale.a', ['swscale'], ['/include', '/include/ffmpeg']),
         DependencyProg('FREETYPE', 'FREETYPE_CONFIG', 'freetype-config', '2.0', ['freetype'], '--ftversion'),
